File: jobs/2026-06-08__12-44-45/migrate_express_ts__8XzGCvp/artifacts/user/helloworld/hello.ts
import { api } from "encore.dev/api";

export const greet = api(
  { method: "GET", path: "/greet/:name", expose: true },
  async ({ name }: { name: string }): Promise<{ message: string }> => {
    return { message: `Hello, ${name}!` };
  }
);

File: jobs/2026-07-02__07-42-24/encore_create_hello_world_go__t52zLq3/artifacts/user/helloworld-zrozs4j0xp/hello/hello.go
package hello

import (
	"context"
)

type Response struct {
	Message string `json:"message"`
}

//encore:api public method=GET path=/hello/:name
func World(ctx context.Context, name string) (*Response, error) {
	return &Response{Message: "Hello, " + name + "!"}, nil
}
